src/main/webapp/resthelp.jsp

```diff
@@ -118,64 +118,162 @@
+        <td>The URL path does not match the expected format.</td>
+        <td><code>/rest/invalidpath/N1</code></td>
+    </tr>
+    <tr>
+        <td>404 (Not Found)</td>
+        <td>Lemma not found for Reference Form: {Reference Form}</td>
+        <td>No matching lemma was found for the provided Reference Form.</td>
+        <td><code>/rest/lemma/UnknownForm</code></td>
+    </tr>
+    <tr>
+        <td>404 (Not Found)</td>
+        <td>More than one Lemma found for Reference Form: {Reference Form}</td>
+        <td>Multiple lemma entries exist for the given Reference Form.</td>
+        <td><code>/rest/lemma/Benedicti</code></td>
+
+    </tr>
+    <tr>
+        <td>404 (Not Found)</td>
+        <td>Lemma not found with ID {id}</td>
+        <td>The requested lemma ID does not exist in the database.</td>
+        <td><code>/rest/item/M9999999</code></td>
+    </tr>
+    <tr>
+        <td>404 (Not Found)</td>
+        <td>Single Reference not found with ID {id}</td>
+        <td>The requested Single Reference ID does not exist in the database.</td>
+        <td><code>/rest/item/B9999999</code></td>
+    </tr>
+    <tr>
+        <td>404 (Not Found)</td>
+        <td>Person not found with ID {id}</td>
+        <td>The requested Person ID does not exist in the database.</td>
+        <td><code>/rest/item/P9999999</code></td>
+    </tr>
+    <tr>
+        <td>404 (Not Found)</td>
+        <td>Source not found with ID {id}</td>
+        <td>The requested Source ID does not exist in the database.</td>
+        <td><code>/rest/item/Q9999999</code></td>
+    </tr>
+    <tr>
+        <td>500 (Internal Server Error)</td>
+        <td>Internal Server Error: {message}</td>
+        <td>An unexpected error occurred while processing the request.</td>
+        <td><code>/rest/item/M1</code> (if database is down)</td>
+    </tr>
+</table>
+
+<h3>Multiple Item Queries</h3>
+<p>The following error scenarios may occur when querying multiple items:</p>
+<table border="1">
+    <tr>
+        <th>Error Code</th>
+        <th>Message</th>
+        <th>Description</th>
+        <th>Example URL</th>
+    </tr>
+    <tr>
+        <td>400 (Bad Request)</td>
+        <td>Invalid request format: {path}</td>
+        <td>The request format is incorrect (missing or malformed identifiers).</td>
+        <td><code>/rest/items/</code></td>
+    </tr>
+    <tr>
+        <td>500 (Internal Server Error)</td>
+        <td>Internal server error due to malformed item identifiers</td>
+        <td>The URL path does not match the expected format for multiple items (e.g., missing commas or invalid characters).</td>
+        <td><code>/rest/items/M6361M6362</code></td>
+    </tr>
+    <tr>
+        <td>500 (Internal Server Error)</td>
+        <td>Internal Server Error: {message}</td>
+        <td>An unexpected error occurred while processing the request.</td>
+        <td><code>/rest/items/M6361,M6362</code> (if database is down)</td>
```

Comment:
There should also be an example what happens when you e.g. use the items endpoint to request 2 items and 1 of them cannot be found (including example JSON result).